index.js

```diff
@@ -15,7 +15,7 @@ const options = {
 	githubToken: process.env.GITHUB_TOKEN,
 	log: console,
 	name: 'Origami Bower Registry',
-	packageDataStore: process.env.PACKAGE_DATA_STORE || 'https://www.ft.com/__origami/service/bower-registry-data',
+	packageDataStore: process.env.PACKAGE_DATA_STORE || 'https://origami-bower-registry-data.ft.com/',
```

Comment:
You can remove the ending slash if you want, we remove it inside the application already -- https://github.com/Financial-Times/origami-bower-registry/blob/54f5ee014a240c4767aaaabffa04bd6bfa0469c7/lib/package-data.js#L46